timescaledb/getting-started/create-hypertable.md

```diff
@@ -81,10 +70,21 @@ effective use of memory and disk resources. By automatically and transparently
 partitioning on time, hypertables improve resource use. Queries and
 data-stores become more efficient.
 
+   COMMENT:
+   I feel like we probably don't need this section but wherever this section is, 
+   I think it would be cool to include a graphic that shows these benefits 
+   https://iobeam.slack.com/archives/C0J94TE4F/p1646682667481189?thread_ts=1646682487.374619&cid=C0J94TE4F
+-->
+
 ## Learn more about hypertables and chunks
 See the [Hypertable How To](/how-to-guides/hypertables) to learn more about
-hypertables and best practices for configuring chunks.
+hypertables and best practices for configuring chunks. You can also learn more 
+about why hypertables help with storing and querying time-series data through 
+the [hypertables and chunks core concepts page][core-concepts-hypertables].
+
+Next, ingest some sample data into TimescaleDB. Continue on to the ['Add time-series data' section][add-data] 
+for information on how to populate the table you just created. 
```

Comment:
```suggestion
Next, ingest some sample stock trade data into TimescaleDB! Step 4, ['Add time-series data' section][add-data], will show you how to populate the hypertable you just created. 
```
Again, open to wording. "Continue on to..." feels wordy to me.

Reply:
```suggestion
Now that you have an empty hypertable, learn how to ingest data. To populate
your hypertable with sample stock trade data, see [Adding time-series data][add-data].
```

"where" is the least important detail so should go last